### DISCUSSION TITLE
pgBouncer configured

### DISCUSSION THREAD

**Discussion by Easy-Cloud-in** [prisma/prisma#15036] [Q&A] [answered]
Hello, can you please tell me if pgBouncer must be configured if I use Prisma in the lambda function? The maximum number of users in my multi-tenant app, where each tenant has its own database, will be only 10 per database.

_Originally posted by @Easy-Cloud-in in https://github.com/prisma/prisma/issues/2443#issuecomment-1225656335_

**Comment by pantharshit00** [comment, marked as answer]
I think you should use Pgbouncer or any other connection pooler for lambda as they are prone for exhausting db connection limit. It depends on the load and amount of connections your database allows. 

You can also use the Prisma Data Proxy here(https://www.prisma.io/data-platform/proxy). It will provide you with a hosted connection pooling and there is also a generous free tier which you can use.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
